spacepy/ctrans/__init__.py

```diff
@@ -136,7 +107,7 @@ class Ellipsoid(dm.SpaceData):
     """Ellipsoid definition class for geodetic coordinates
 
     Other Parameters
-    ----------------
+    ================
```

Comment:
Can we revert all these without it being a complete disaster? [These should all be -,](https://numpydoc.readthedocs.io/en/latest/format.html) and we don't need to change everything existing, but we should change in the wrong direction.z

Reply:
This one may be a little harder, I think the easiest and cleanest way to do it would be to change it back in a commit, I have a commit for all these changes anyways, so I can just lump it into there. Hopefully because I'm reverting it back to what it was before it was changed, that the diff will recognize that, and it will remove it from the diff completely.

Reply:
The github diff will look clean, but the history is still hard to crawl through. I think you're doing the rebasing at this point so I think we're good to go.

Reply:
Yes I got it down, grinding out the changes now

Reply:
I think I hit all of these, I think that this is resolved.